src/main/java/com/google/cloud/tools/libraries/libraries.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0">
+<!-- Warning: non-normative. Probably not even correct. -->
+<apis>
+
+  <api name="BigQuery API v2"
+      definition="https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest"
+      documentation="https://cloud.google.com/bigquery/docs/reference/rest/v2/"
+    >
+    
+    <client language="java"
+      site="https://developers.google.com/api-client-library/java/apis/bigquery/v2"
+      apireference="https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/"
+      revision="352"
+      version="1.22.0"
+      status="release"    
+      source="https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigquery"
+      languageLevel="1.7.0"
+      status="stable">
+        <dependency xmlns="http://maven.apache.org/POM/4.0.0">
+          <groupId>com.google.apis</groupId>
+          <artifactId>google-api-services-bigquery</artifactId>
+          <version>v2-rev352-1.22.0</version>
+        </dependency>
+        <platforms>
+          <platform>Android</platform>
+          <platform>AES-7</platform>
+          <platform>AES-8</platform>
+          <platform>Java 7</platform>
+        </platforms>
+    </client>
+      
+    <client language="c#"
+      site="https://developers.google.com/api-client-library/dotnet/apis/bigquery/v2"
+      apireference="https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/csharp/latest/"
+      status="release"    
+      source="https://github.com/GoogleCloudPlatform/google-cloud-dotnet/tree/master/apis/Google.Cloud.BigQuery.V2"
+      languageLevel="4.0"
+      package="Google.Apis.bigquery"
```

Comment:
So this is the NuGet package name?  Would we want a `version` here too?  (And if not, why have `version` and `revision` in the `java` client section above?)